<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ITS0568.local" failures="0" tests="3" name="categoryNavigation.validateNavigationTest" time="11.006" errors="1" timestamp="20 May 2020 15:55:54 GMT" skipped="1">
  <testcase classname="categoryNavigation.validateNavigationTest" name="mainPageNavValidation" time="10.986"/>
  <testcase classname="categoryNavigation.validateNavigationTest" name="pdpValidation" time="0.020">
    <error message="Method categoryNavigation.validateNavigationTest.pdpValidation() didn&#039;t finish within the time-out 10" type="org.testng.internal.thread.ThreadTimeoutException">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method categoryNavigation.validateNavigationTest.pdpValidation() didn't finish within the time-out 10
]]>
    </error>
  </testcase> <!-- pdpValidation -->
  <testcase classname="categoryNavigation.validateNavigationTest" name="filterValidation">
    <skipped/>
  </testcase> <!-- filterValidation -->
</testsuite> <!-- categoryNavigation.validateNavigationTest -->
